src/containers/SemesterListContainer.jsx

```diff
@@ -0,0 +1,30 @@
+import { connect } from 'react-redux';
+import SemesterList from '../components/SemesterList';
+
+const shortenYear = (year) => {
+  const shortYearNumber = year % 100;
+  const shortYearString = shortYearNumber.toString();
+  if (shortYearString.length < 2) {
+    return '0' + shortYearString;
+  }
+  return shortYearString;
+};
+
+const makeSemesterList = (entryYear) => {
```

Comment:
This doesn't make sense in the context of students who take LOAs or graduate early. I feel like this list should get created when students declare that they have taken courses, rather than being generated the same way every time. Can we just have semesters 1-8 by default, instead of these names?